openmc/element.py

```diff
@@ -164,7 +212,14 @@ def expand(self, percent, percent_type, enrichment=None,
                 abundances[nuclide] = NATURAL_ABUNDANCE[nuclide]
 
         # Modify mole fractions if enrichment provided
-        if enrichment is not None:
+        # Old treatment for Uranium
+        if enrichment is not None and enrichment_target is None:
```

Comment:
I wonder if this condition should be extended to include the case where enrichment_target=='U235'?

Currently if anyone tried using this function for uranium and explicitly set the enrichment_target to U235 it would raise an error, because uranium has more than 2 natural isotopes. The error message will say that uranium can't be enriched with the inbuilt procedure, but of course that is not correct.